### DISCUSSION TITLE
login help. please

### DISCUSSION THREAD

**Discussion by Wrath0101** [runelite/runelite#17214] [Q&A]
I cant get rune light to work. it keeps saying that my email and password is wrong but it isn't. does anyone know how i can solve it or perhaps a way that I can sign in before it try's connecting me to a sever.   EDIT... I've tried it on multiple pcs and even tried making a new account but to no avail

**Comment by Felanbird** [comment]
If your username & password are correct, you likely have a Jagex account and need to start Runelite from the [Jagex Launcher](https://www.jagex.com/en-GB/launcher)

If you continue to have issues, you'll likely need to join the [oldschool runescape discord](https://discord.gg/osrs), as Runelite has nothing to do with accounts.
